### DISCUSSION TITLE
Official IM chat channel: #pycontribs on libera.chat

### DISCUSSION THREAD

**Discussion by ssbarnea** [pycontribs/jira#1055] [Show and tell]
If anyone needs attention for reviewing or merging a pull request, please use `#pycontribs` irc channel on freenode.chat network. This channel also gets notifications about new PRs.

Please, do not ping cores about pull requests that did not pass CI/CD, wait for them to report green before doing that.

https://web.libera.chat/?channel=#pycontribs

**Comment by adehad** [comment]
I am most active on https://matrix.to/#/#pycontribs:matrix.org, so also reach out to us there!
